com.unity.render-pipelines.universal/Shaders/Nature/SpeedTree8.shader

```diff
@@ -54,6 +54,7 @@ Shader "Universal Render Pipeline/Nature/SpeedTree8"
             #pragma multi_compile _ _MAIN_LIGHT_SHADOWS _MAIN_LIGHT_SHADOWS_CASCADE _MAIN_LIGHT_SHADOWS_SCREEN
             #pragma multi_compile _ _ADDITIONAL_LIGHTS_VERTEX _ADDITIONAL_LIGHTS
             #pragma multi_compile _ _ADDITIONAL_LIGHT_SHADOWS
+            #pragma multi_compile_fragment _ _REFLECTION_PROBE_BLENDING
```

Comment:
Is it intentional that there is no box projection for this?

Reply:
Yes, the idea is that terrain and speed tree shaders are generally used outdoors. For which box projection isn't very useful. By removing box projection here we might save a few cycles and just blend as if it were a normal probe.

Reply:
Is this something that's done elsewhere? I could imagine having terrain and trees indoors.

Reply:
We tried to implement the box projection for the terrain, but the formula to calculate the box projection can't handle uneven terrain.
![image](https://user-images.githubusercontent.com/4364279/114548798-85c4e900-9c60-11eb-9eb1-e1303e63cc3c.png)
The lines in the terrain are pillars projected from the left side of the scene. And the terrain goes uphill from the right to the left.

The same formula is used for box projection in built-in so I expect the same issue to appear, but it seems like they do allow box projection.

Reply:
I think my take on this is that it should be on the tech artist to know not to use box projection in this case, but I agree that for terrain it makes some sense not to support it for now (until we have tiling), but for speedtree, I think we should add it for now.

Reply:
Enabled box projection for SpeedTree8 in 742303f. Terrain still has it disabled.